services/watson-extension/src/watson_extension/clients/general/redhat_status.py

```diff
@@ -0,0 +1,27 @@
+import abc
+import injector
+import logging
+from aiohttp import ClientResponse
+from watson_extension.clients.platform_request import AbstractPlatformRequest
+
+
+logger = logging.getLogger(__name__)
+
+
+class RedhatStatusClient(abc.ABC):
+    @abc.abstractmethod
+    async def check_services_offline(self) -> ClientResponse: ...
+
+
+class RedhatStatusClientHttp(RedhatStatusClient):
+    def __init__(
+        self,
+        platform_request: injector.Inject[AbstractPlatformRequest],
```

Comment:
Was it easier to use the platform request obj instead of using aiohttp directly?

```
try:
            async with aiohttp.ClientSession() as session:
                async with session.get(
                    "https://status.redhat.com/api/v2/incidents/unresolved.json"
                ) as status_response:
                    result = await status_response.json()
        except Exception as e:
            print(
                f"An Exception occured while handling response from status.redhat.com: {e}"
            )
        ```

Reply:
I think they are basically doing the same thing underneath the hood. Used platform request for the sake of consistency. Also, I think the platform_request interface is much simpler.